README.md

```diff
@@ -1,8 +1,64 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/core_java)]:
+      
+  - Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DavidGordon/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**Flashtyper type tester** [[GitHub](https://github.com/davidgordon12/FlashTyper-MVC)]: Online typing speed tracker with a global leaderboard. Written in C# ASP.NET, data is stored in a Microsoft SQL Server database, hosted in the cloud via Azure
+
+**Epistle** [[GitHub](https://github.com/davidgordon12/epistle)]: Note-taking web application focused on simplicity and portability. Allows the user to create 'Bookshelves' to store and organize their notes. Implemented in ASP.NET Core, EntityFrameworkCore and Microsft SQL Server database.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2023-present)**: As a profesional developer at Jarvis, I am responsible for creating and managing various software projects, as well as testing, debugging, automating, and deploying the applications. I use Agile/Scrum methods to keep my work organized and ensure I finish tasks on time, and proper GitFlow to keep my software workflow consistent.
+
+**Software Developer, VAR Lab (2023 Jan-2023 April)**: Designed software solutions and tests to support the creation of an augmented/virtual reality platform that can be easily extended/modified to support a range of demo projects in support of ongoing Applied Research objectives at Conestoga College.
+
+
+## Education
+**Conestoga College (2020-2023)**, Diploma of Computer Programming, Computer Engineering
+- Scholarship
+- Dean's List (2015, 2016): Ut enim ad minim veniam
+- GPA: 3.8/4.0
+
+**XYZ university (2016-2018)**, Master of Engineering, Electrical and Computer Engineering
+
+
+## Miscellaneous
```

Comment:
I believe these are all placeholders. Unless you have nothing that makes you an individual, you should be taking this opportunity to show why you're awesome.